pkg/cable/vxlan/vxlan.go

```
@@ -86,13 +86,17 @@ func init() {
 	cable.AddDriver(CableDriverName, NewDriver)
 }
 
-func NewDriver(localEndpoint types.SubmarinerEndpoint, localCluster types.SubmarinerCluster) (cable.Driver, error) {
+func NewDriver(localEndpoint *types.SubmarinerEndpoint, localCluster *types.SubmarinerCluster) (cable.Driver, error) {
+	if localEndpoint == nil || localCluster == nil {
+		return nil, fmt.Errorf("NewDriver called with nil localEndpoint and/or localCluster")
+	}
```

Let's let it panic here too...